controller/update.go

```diff
@@ -144,6 +144,8 @@ func (c controller) addFirst(ctx context.Context, req Request, opts UpdateOption
 }
 
 func (c controller) runAddWorker(ctx context.Context, req Request, opts UpdateOptions) (Request, error) {
+	c.Config.Logger.Info(nil, "controller: adding units %v", req.SliceIDs)
```

Comment:
Please consider reading https://github.com/giantswarm/inago/pull/235#issuecomment-209897009

I thought it is not necessary for info logging. I also think it is ugly as fuck. I am totally open on that. What would you like to see?